app/src/main/java/org/mozilla/fenix/settings/studies/StudiesFragment.kt

```diff
@@ -40,8 +39,7 @@ class StudiesFragment : Fragment() {
             interactor,
             requireContext().settings(),
             experiments,
-            ::isAttached,
-            requireContext().metrics
+            ::isAttached
```

Comment:
nit: there was an recent agreement to keep the comma on the last parameter to minimize diff.  This is after considering that the new Kotlin style promotes keeping commas.